apps/web-e2e/src/e2e/posts.cy.ts

```diff
@@ -310,7 +310,7 @@ describe('posts', () => {
     });
   });
 
-  it('can tag and remove tags from existing post', () => {
+  it.only('can tag and remove tags from existing post', () => {
```

Comment:
```suggestion
  it('can tag and remove tags from existing post', () => {
```
@catch-21, I think you forgot to remove this `.only` from this test, right?